portlets/src/main/webapp/vue-app/engagement-center/components/challenges/ChallengeCard.vue

```diff
@@ -99,37 +75,40 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
             </div>
           </div>
         </div>
-        <div class="font-weight-bold tertiary--text text-subtitle-2 mb-auto">
-          {{ challenge && challenge.points }} {{ $t('challenges.label.points') }}
+        <div class="my-auto mx-auto">
+          <v-tooltip bottom>
+            <template #activator="{ on, attrs }">
+              <span
+                class="d-flex-inline position-relative text-truncate-2 font-weight-bold text--secondary" 
+                v-bind="attrs"
+                v-on="on">
+                {{ challengeTitle }}
+              </span>
+            </template>
+            {{ challengeTitle }}
+          </v-tooltip>
         </div>
-        <div :class="isMobile && 'flex-column'" class="d-flex">
-          <div :class="isMobile && 'py-1'" class="d-flex flex-row">
-            <v-icon size="16" class="primary--text ps-1">fas fa-calendar-day</v-icon>
-            <span class="my-auto ms-2" v-sanitized-html="DateInfo"></span>
-          </div>
+        <div :class="isMobile && 'flex-column ms-1'" class="d-flex">
```

Comment:
Please avoid using JS for Style purpose

```suggestion
        <div class="d-flex flex-column flex-sm-row ms-1 ms-sm-0">
```

Reply:
changes applied !